libs/domain/cart/entry/src/entry.component.ts

```diff
@@ -99,7 +99,8 @@ export class CartEntryComponent
   protected $productLink = computed(() => {
     return this.semanticLinkService.get({
       type: RouteType.Product,
-      id: this.$product()?.sku,
+      id: featureVersion <= '1.4' ? this.$product()?.sku : undefined,
```

Comment:
```suggestion
      id: featureVersion <= '1.4' ? this.$product()?.sku : undefined,
```

I assume this should be `< `1.4` ?